Tests/UnitTests/Paywalls/Events/PaywallEventsManagerTests.swift

```diff
@@ -199,8 +199,7 @@ class PaywallEventsManagerTests: TestCase {
         let task1 = Task<Int, Error> { [manager = self.manager!] in try await manager.flushEvents(count: 1) }
         let task2 = Task<Int, Error> { [manager = self.manager!] in try await manager.flushEvents(count: 1) }
 
-        let result1 = try await task1.value
-        let result2 = try await task2.value
+        let (result1, result2) = try await (task1.value, task2.value)
```

Comment:
not entirely sure that this fixes it? but I did want to ensure that we don't run one and _then_ the other